webapp/graphite/render/hashing.py

```diff
@@ -33,11 +33,11 @@ def hashRequest(request):
   return compactHash(normalizedParams)
 
 
-def hashData(targets, startTime, endTime):
+def hashData(targets, startTime, endTime, node):
```

Comment:
You also need to fix tests in https://github.com/graphite-project/graphite-web/blob/0.9.x/webapp/tests/test_render.py#L97-L102 btw - just humble reminder. :)